pkg/internal/checkup/checkup.go

```diff
@@ -86,14 +86,18 @@ func New(client kubeVirtVMIClient, namespace string, checkupConfig config.Config
 }
 
 func (c *Checkup) Setup(ctx context.Context) (setupErr error) {
+	const setupTimeout = 10 * time.Minute
```

Comment:
Are you sure 10 minutes are OK even if the images are not yet pulled?

Reply:
We can increase it to 15 mins.
WDYT?

Reply:
I'm not sure really, we need to measure it and give some extra to be sure. 
It's possible that 10 minutes is enough

Reply:
It worked for me in the 10 minutes time frame (including image pulling).

Reply:
OK then that's enough for now. 10 minutes should be more than enough.